Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "aardwolf"
 version = "0.1.0"
 authors = ["Eric Chadbourne <sillystring@protonmail.com>", "Elijah Mark Anderson <kd0bpv@gmail.com>", "Paul Woolcock <paul@woolcock.us>", "Asonix <asonix.dev@gmail.com>"]
 description = "Powering connected social communities with free software."
+autobins = false
```

Comment:
sneaking in that `autobins = false` to avoid compilation warnings and be future-proof